### PR TITLE
fix(security) fix hotlink on media

### PR DESCRIPTION
**Pull request by loiclau**
## Description

protect media

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [x] 19.04.x
- [x] 19.10.x
- [x] 20.04.x
- [x] 20.10.x (master)

<h2> How this pull request can be tested ? </h2>

## Checklist

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by sc979 — DISMISSED** [review]

**Review by kduret — DISMISSED** [review]
LGTM
But we'll need to check every possible formats
I don't know if we can import `.PNG` (uppercase)

File: www/img/media/.htaccess [hunk 1]
```diff
@@ -0,0 +1,6 @@
+RewriteEngine On
+Options +FollowSymLinks
+Options +SymLinksIfOwnerMatch
+
+RewriteCond %{HTTP_HOST}@@%{HTTP_REFERER} !^([^@]*)@@https?://\1/.*
+RewriteRule .*\.(gif|GIF|jpg|JPG|jpeg|bmp|BMP|mpeg|png)$ - [F]
```
Comment (loiclau) [inline www/img/media/.htaccess:6]:
```suggestion
RewriteRule .*\.(jpg|jpeg|png|gif||gd2)$ - [F]
```

Comment (loiclau) [reply www/img/media/.htaccess]: https://github.com/centreon/centreon/blob/master/www/include/options/media/images/syncDir.php#L66

**Review by sc979 — DISMISSED** [review]

File: www/img/media/.htaccess [hunk 2]
```diff
@@ -0,0 +1,6 @@
+RewriteEngine On
+Options +FollowSymLinks
+Options +SymLinksIfOwnerMatch
+
+RewriteCond %{HTTP_HOST}@@%{HTTP_REFERER} !^([^@]*)@@https?://\1/.*
+RewriteRule .*\.(jpg|jpeg|png|gif||gd2)$ - [F]
```
Comment (kduret) [inline www/img/media/.htaccess:6]:
```suggestion
RewriteRule .*\.(jpg|jpeg|png|gif|gd2)$ - [F]
```

**Review by kduret — CHANGES_REQUESTED** [review]

**Review by kduret — APPROVED** [review]

**Review by sc979 — APPROVED** [review]

**Pull request merged**